docs/ascension/compatibility.mdx

```diff
@@ -11,13 +11,15 @@ DiscordSRV requires Java 8 or newer
 
 ## Server Software compatibility
 
-| Server software       | Supported versions                                   |
-|-----------------------|------------------------------------------------------|
-| [Bukkit], [Spigot], [Paper] | ✅ Minecraft 1.8.8 and up<br/>⚠️ Minecraft 1.7.10[^1] |
-| [BungeeCord], [Waterfall] | ✅ Minecraft 1.7.10 and up                            |
-| [Velocity]              | ✅ 3.0.0 and up<br/>❌ 2.0.0 and below                 |
+| Server software       | Supported versions                                   | Available For Testing? |
+|-----------------------|------------------------------------------------------|------------------------|
+| [Bukkit], [Spigot], [Paper] | ✅ Minecraft 1.8.8 and up<br/>⚠️ Minecraft 1.7.10[^1] | Yes |
+| [BungeeCord], [Waterfall] | ✅ Minecraft 1.7.10 and up                            | No |
+| [Velocity]              | ✅ 3.0.0 and up<br/>❌ 2.0.0 and below                 | No |
+| [Fabric]                | ✅ 1.21.4<br/>⚠️1.21 - 1.21.3[^2]<br/>❌ 1.20.4 and below[^2] | Yes |
 
 [^1]: Requires [updating ASM](#) <!-- TODO: fix anchor (faq#asm-update) -->
+[^2]: These versions have not been extensively tested. Please report in #ascension-testing in our Discord server if you can use the plugin on them without issue.
```

Comment:
I would rather not handle posts regarding those versions as they aren't that important (non-latest patches), please remove